Property 'parameterRef' is missing in type 'SopsStringParameter' but required in type 'IStringParameter'

When using the new CDK lib `2.227.0` the efollowing error occures:

```
node_modules/cdk-sops-secrets/lib/SopsStringParameter.d.ts(42,22): error TS2420: Class 'SopsStringParameter' incorrectly implements interface 'IStringParameter'.
  Property 'parameterRef' is missing in type 'SopsStringParameter' but required in type 'IStringParameter'.
```

--- a/.projenrc.js
+++ b/.projenrc.js
@@ -37,7 +37,7 @@ const actions_TestBuild = [
 const project = new awscdk.AwsCdkConstructLibrary({
   author: 'Markus Siebert',
   authorAddress: 'markus.siebert@deutschebahn.com',
-  cdkVersion: '2.224.0',
+  cdkVersion: '2.227.0',
   majorVersion: 2,
   stability: 'stable',
   homepage: 'https://constructs.dev/packages/cdk-sops-secrets',

--- a/API.md
+++ b/API.md
@@ -699,6 +699,7 @@ Any object.
 | <code><a href="#cdk-sops-secrets.SopsStringParameter.property.stack">stack</a></code> | <code>aws-cdk-lib.Stack</code> | The stack in which this resource is defined. |
 | <code><a href="#cdk-sops-secrets.SopsStringParameter.property.stringValue">stringValue</a></code> | <code>string</code> | The parameter value. |
 | <code><a href="#cdk-sops-secrets.SopsStringParameter.property.sync">sync</a></code> | <code><a href="#cdk-sops-secrets.SopsSync">SopsSync</a></code> | *No description.* |
+| <code><a href="#cdk-sops-secrets.SopsStringParameterProps.property.parameterRef">parameterRef</a></code> | <code>string</code> | A reference to a Parameter resource. |
 
 ---
 

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "@typescript-eslint/eslint-plugin": "^8",
     "@typescript-eslint/parser": "^8",
     "aws-cdk": "^2",
-    "aws-cdk-lib": "2.224.0",
+    "aws-cdk-lib": "2.227.0",
     "commit-and-tag-version": "^12",
     "constructs": "10.0.5",
     "eslint": "^9",
@@ -82,7 +82,7 @@
     "typescript": "^4.9.5"
   },
   "peerDependencies": {
-    "aws-cdk-lib": "^2.224.0",
+    "aws-cdk-lib": "^2.227.0",
     "constructs": "^10.0.5"
   },
   "dependencies": {

--- a/src/SopsStringParameter.ts
+++ b/src/SopsStringParameter.ts
@@ -2,6 +2,7 @@ import { Grant, IGrantable } from 'aws-cdk-lib/aws-iam';
 import { IKey } from 'aws-cdk-lib/aws-kms';
 import {
   IStringParameter,
+  ParameterReference,
   ParameterTier,
   StringParameter,
 } from 'aws-cdk-lib/aws-ssm';
@@ -56,6 +57,7 @@ export class SopsStringParameter extends Construct implements IStringParameter {
   readonly parameterName: string;
   readonly parameterType: string;
   readonly stringValue: string;
+  readonly parameterRef: ParameterReference;
 
   public constructor(
     scope: Construct,
@@ -82,6 +84,7 @@ export class SopsStringParameter extends Construct implements IStringParameter {
     this.parameterName = this.parameter.parameterName;
     this.parameterType = this.parameter.parameterType;
     this.stringValue = this.parameter.stringValue;
+    this.parameterRef = this.parameter.parameterRef;
 
     this.sync = new SopsSync(this, 'SopsSync', {
       encryptionKey: this.parameter.encryptionKey,

